src/review/routes/homepage/Homepage.js

```diff
@@ -1,13 +1,23 @@
 import React, { PropTypes } from 'react'
 import { connect } from 'react-redux'
 import ProductHome from 'review/routes/product/Home'
+import Error from 'review/routes/Error'
 import Dashboard from 'review/routes/dashboard/Dashboard'
 
-function Homepage(props) {
-  return props.user ? <Dashboard /> : <ProductHome />
+function Homepage({ user, error }) {
```

Comment:
Actually, this pattern is breaking https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md.